lms/static/js/simplewiki-AutoSuggest_c_2.0.js

```diff
@@ -1,961 +0,0 @@
-/**
- *  author:		Timothy Groves - http://www.brandspankingnew.net
- *	version:	1.2 - 2006-11-17
- *              1.3 - 2006-12-04
- *              2.0 - 2007-02-07
- *
- */
-
-var useBSNns;
```

Comment:
This was a component of django-simple-wiki. We collaborated with the author of django-simple-wiki to improve it, and in the process, django-simple-wiki became django-wiki, which is what we're using today. 

I expect all simple-wiki files are obsolete and should be killed.